### PR TITLE
Integrations Health dashboard bug fix- show errors right after fresh install

### PR DESCRIPTION
**Pull request by tallieber**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/34129

## Description
After fresh install of 6.1 and installing the incidents and integrations health check pack, integrations health dashboard shows 1 failed incident rather than 0 and in the failed instance information show error.
getList command return "Item not found" in the content field when no list was created (on fresh install), caused problem in the different scripts that expected to get empty field instead.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/IntegrationsAndIncidentsHealthCheck/pack_metadata.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "Integrations & Incidents Health Check",
     "description": "Do you know which of your integrations or open incidents failed? With this content, you can view your failed integrations and open incidents",
     "support": "xsoar",
-    "currentVersion": "1.1.15",
+    "currentVersion": "1.2.0",
```
Comment (bakatzir) [inline Packs/IntegrationsAndIncidentsHealthCheck/pack_metadata.json:5]: bugfix should be a revision usually

File: Packs/IntegrationsAndIncidentsHealthCheck/ReleaseNotes/1_2_0.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (bakatzir) [inline Packs/IntegrationsAndIncidentsHealthCheck/ReleaseNotes/1_2_0.md:1]: please follow https://confluence.paloaltonetworks.com/display/DemistoContent/Release+Notes+And+Descriptions

File: Packs/IntegrationsAndIncidentsHealthCheck/pack_metadata.json (hunk 1 above)
Comment (tallieber) [reply Packs/IntegrationsAndIncidentsHealthCheck/pack_metadata.json:5]: I ran again with revision, also bumped to 1.2.0

Comment (tallieber) [reply Packs/IntegrationsAndIncidentsHealthCheck/pack_metadata.json:5]: I ran again with revision, also bumped to 1.2.0

**Comment by tallieber** [comment]
@bakatzir list_json = None return chart with N/As while list_json = {} return empty chart. What is preferred ?
image or 
image

**Review by bakatzir — APPROVED** [review]

File: Packs/IntegrationsAndIncidentsHealthCheck/ReleaseNotes/1_2_0.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+#### Scripts
+##### IntegrationsCheck-Widget-NumberFailingInstances
+- Fix an issue with IntegrationsErrorsInfo widget that shows error after fresh install.
```
Comment (amshamah419) [inline Packs/IntegrationsAndIncidentsHealthCheck/ReleaseNotes/1_2_0.md:4]:
```suggestion
- Fix an issue where the *IntegrationsErrorsInfo* widget would show an error after a fresh install.
```

File: Packs/IntegrationsAndIncidentsHealthCheck/ReleaseNotes/1_2_0.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+##### IntegrationsCheck-Widget-IntegrationsErrorsInfo
+- Fix an issue with  NumberFailingInstances widget that shows 1 failed instance after fresh install while there is no fail instances.
```
Comment (amshamah419) [inline Packs/IntegrationsAndIncidentsHealthCheck/ReleaseNotes/1_2_0.md:6]:
```suggestion
- Fix an issue where the *NumberFailingInstances* widget would display 1 failed instance after a fresh install when there were no failed instances. 
```

**Pull request merged**